src/App.tsx

```diff
@@ -44,6 +44,16 @@ function App() {
     textarea
   );
   const setTextToParseThrottle = useThrottleCallback(setTextToParse, 2);
+  const inputRef = useRef({} as HTMLTextAreaElement);
```

Comment:
It's probably safer to assign the input ref using a callback function like 
`const setBoxRef = (boxElement: HTMLTextAreaElement) => boxRef.current = boxElement;`
rather than coercing an empty object to the type  HTMLTextAreaElement